hack/custom_rpms.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+set -e
+
+. $(dirname "$0")/common.sh
+
+RPMS_SRC="${RPMS_SRC:-}"
+REMOVE_PACKAGES="${REMOVE_PACKAGES:-}"
+RPMS_NODE_ROLE="${RPMS_NODE_ROLE:-node-role.kubernetes.io/worker}"
+
+if [ "$RPMS_SRC" == "" ]; then
+	echo "[ERROR]: No RPMS_SRC provided"
+	exit 1
+fi
+
+cat <<EOF | oc apply -f -
+kind: ConfigMap
```

Comment:
can we have the ConfigMap and DaemonSet be their own files, maybe under features-config, like any other feature?

Reply:
the environment variables (RPMS_SRC and REMOVE_PACKAGES) can be env in the daemon set, then they can be easily overriden by kustomize.

hmm, that wont fly with your changes detection mechanism (cause you count on the configmap to change)
so maybe create an additional ConfigMap with just these two?